### PR TITLE
Add RC ready wait to CL

### PR DESCRIPTION
**Pull request by sjug**
Add another possible object to wait for and the end of the test.

### CODE REVIEW

File: test/extended/cluster/cl.go [hunk 1]
```diff
@@ -202,6 +202,26 @@ var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
 
 		// Wait for builds and deployments to complete
 		for _, ns := range namespaces {
+			rcList, err := oc.AdminKubeClient().CoreV1().ReplicationControllers(ns).List(metav1.ListOptions{})
+			if err != nil {
+				e2e.Logf("Error listing RCs: %v", err)
+			}
+			rcCount := len(rcList.Items)
+			if rcCount > 0 {
+				e2e.Logf("Waiting for %d RCs in %s", rcCount, ns)
```
Comment (jmencak) [inline test/extended/cluster/cl.go]: in *namespace* %s ?

File: test/extended/cluster/cl.go [hunk 2]
```diff
@@ -202,6 +202,26 @@ var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
 
 		// Wait for builds and deployments to complete
 		for _, ns := range namespaces {
+			rcList, err := oc.AdminKubeClient().CoreV1().ReplicationControllers(ns).List(metav1.ListOptions{})
+			if err != nil {
+				e2e.Logf("Error listing RCs: %v", err)
+			}
+			rcCount := len(rcList.Items)
```
[the remark this answers is not in the document]
Comment (sjug) [reply test/extended/cluster/cl.go]: Yeah probably, but the length of a nil slice is 0, so it should be fine?

**Comment by sjug** [comment]
@jmencak Reformatted `WaitForRCReady()` like your `SyncPods()`

File: test/extended/cluster/cl.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (jmencak) [reply test/extended/cluster/cl.go]: I was referring to a possible runtime panic when dereferencing a nil pointer when `rcList == nil`, hence the question whether `rcList` can ever be nil.

**Comment by jmencak** [comment]
/hold
/lgtm
Feel free to remove hold if you're sure `rcList != nil` always holds. I'd prefer better error checking in the code though.

File: test/extended/cluster/cl.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (sjug) [reply test/extended/cluster/cl.go]: Failing in the error condition now. Thanks Jiri!

**Pull request merged**